baseframe/static/js/baseframe-material.js

```diff
@@ -249,6 +249,26 @@ window.Baseframe.Forms = {
         }
       }
     })
+  },
+  showValidationErrors: function(formId, errors) {
+    Object.keys(errors).forEach(function(fieldName) {
+      if (Array.isArray(errors[fieldName])) {
```

Comment:
Is there a chance that `errors[fieldName]` would _not_ be an array?

Reply:
Yes, if the error is not due a form validation error.
In case of hgtv, it could be other errors in the form like logo upload error.